scripts/ctest/ctest_driver.py

```diff
@@ -108,12 +148,19 @@ def run_single_test_suite(suite, ctest_path, cmake_build_path, build_config, dis
     if repeat:
         # Run the tests multiple times. Previous test results are deleted, new test results are combined in a file per
         # test runner.
-
         test_result_prefix = 'Repeat'
         repeat = int(repeat)
+        total_runs = repeat
```

Comment:
Shouldn't total start at zero?

Reply:
fixed

Reply:
Wait no, it's supposed to be set to repeat. total_runs is the number of runs that have been ran during the repeat loop. This is separate from the repeat variable because it can be different with the soft timeout